filament/backend/src/vulkan/VulkanBinder.h

```diff
@@ -170,7 +170,8 @@ class VulkanBinder {
         VkShaderModule shaders[SHADER_MODULE_COUNT]; // 8*2 bytes
         RasterState rasterState; // 248 bytes
         VkRenderPass renderPass; // 8 bytes
-        VkPrimitiveTopology topology; // 4 bytes
+        VkPrimitiveTopology topology : 16; // 2 bytes
+        uint16_t subpass; // 2 bytes
```

Comment:
is this the same "subpass" value than above?  Maybe it should be its own typedef then?

Reply:
Let me rename this to `subpassIndex` too, that's really what it is.